app.py

```diff
@@ -6,14 +6,17 @@
 from flask import Flask, request, jsonify
 from flask_cors import CORS
 
+# loading the model
+summarizer = 'modelfile.pkl'
+model = p.load(open(summarizer, 'rb'))
```

Comment:
we need the model loaded before calling any route or they will fail

Reply:
That is what I did.